.github/workflows/ci.yml

```
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11-dev"]
```

```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11-dev - 3.11"]
```
Can we see if this works - It was suggested to me on black's CI.

---

```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11.0-rc - 3.11"]
```